src/View3d.ts

```diff
@@ -507,7 +507,7 @@ export class View3d {
    * @param {number=} oh Ignored.
    * @param {Object=} eOpts Ignored.
    */
-  resize(comp: HTMLElement | null, w: number, h: number, ow?: number, oh?: number, eOpts?: unknown): void {
+  resize(comp: HTMLElement | null, w?: number, h?: number, ow?: number, oh?: number, eOpts?: unknown): void {
```

Comment:
I wonder if the signature for this could just be something like HTMLResizeEvent... Is this function directly invoked from window.addEventListener('resize'...) ?

Reply:
Looks like it goes through an arrow function on that event listener (View3d.ts:62). cell-viewer was calling it without arguments and creating a type error; I added `null` to fill in that first argument, but am thinking now that maybe I should just have made all arguments optional.